### DISCUSSION TITLE
Do not break on error during the "watch"

### DISCUSSION THREAD

**Discussion by akhansari** [nushell/nushell#14644] [Q&A] [answered]
Is there any possibility to not break the watcher if there are any errors.
This could be very useful for instance to continuously run unit tests.

Like:
```nu
watch . --no-break --glob=**/*.ts { clear; pnpm test }
```

**Comment by Lockszmith-GH** [comment, marked as answer]
you can encapsulate the pnpm call in a try catch, and have the catch display the error without failing the call

**Reply by akhansari** [reply]
Oh thanks! TIL

```nu
watch . --glob=**/*.ts { clear; try { pnpm test } }
```
